tests/test_app.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from app import create_app, db
+from app.models import User
+
+@pytest.fixture
+def app():
+    app = create_app(testing=True)
+
+    with app.app_context():
+        db.create_all()
+
+    yield app
+
+    with app.app_context():
+        db.drop_all()
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
+
+def test_create_user(client):
+    user_data = {
```

Comment:
you might consider making this a fixture. repeating all these values as strings in your assertions feels cumbersome, and I feel you're gonna need some of this stuff soon enough once you add login tests, etc.

Could save that for the next PR though!